java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -733,6 +735,8 @@ private static Object[] streamCommands(BaseTransaction<?> transaction) {
                 .xgroupCreate(streamKey1, groupName1, "0-0")
                 .xgroupCreate(
                         streamKey1, groupName2, "0-0", StreamGroupOptions.builder().makeStream().build())
+                .xgroupCreateConsumer(streamKey1, groupName1, consumer1)
+                .xgroupDelConsumer(streamKey1, groupName1, consumer1)
```

Comment:
Add todo for XREADGROUP to ensure transaction DelConsumer returns > 0?

Reply:
Yes.  That is coming.